customized_list_view/models/ir_module.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 ooops404
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models
+
+
+class Module(models.Model):
+    _inherit = "ir.module.module"
+
+    def _button_immediate_function(self, function):
```

Comment:
**praise** this is an elegant way of making the changes persistent

**thought**: not all changes may be applied. at any given time. I'd say we could either add the `active` field to the `custom.list.view` model, or another boolean field to keep track of which changes need to be applied and which ones should not.